config/config/wc-config.yaml

```diff
@@ -252,3 +255,18 @@ networkPolicies:
 
   alertmanager:
     enabled: true
+
+## Open policy agent configuration
+opa:
```

Comment:
We have a task for moving gatekeeper config under `gatekeeper` instead of `opa` so I would suggest making that change already here
```suggestion
gatekeeper:
```

Reply:
I changed the name and I hope I found every one.